src/systemjs-webpack-interop.d.ts

```diff
@@ -6,7 +6,9 @@ declare module "systemjs-webpack-interop" {
   export function modifyWebpackConfig(
     webpackConfig: WebpackConfig
   ): WebpackConfig;
-  export function verifyWebpackConfig(webpackConfig): void;
+  export function verifyWebpackConfig(
```

Comment:
The name of the function is `checkWebpackConfig` - please fix (even though I know you didn't add it incorrectly, it was already there)